contextualized/regression/tests.py

```diff
@@ -83,8 +83,29 @@ def _quicktest(self, model, univariate=False, correlation=False):
         err_trained = ((y_true - y_preds)**2).mean()
         assert err_trained < err_init, "Model failed to converge"
 
+    def test_workers(self):
```

Comment:
I removed some extra tests here since the dataloader will perform the same for any of the classes that extend the ContextualizedRegressionBase class. I should note that these are really slow though, they take 90% of the whole testing time and are just 1 of 11 tests. On my machine, the multi-worker training does about 7 iterations per second, versus about 700 without multiple workers. 

We aren't duplicating data so this isn't time spent copying data, and I wouldn't expect communication costs to create a 100x slowdown. Would you mind looking into this?